File: DroneManager.API/src/UTMService/Models/Track.java
package UTMService.Models;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 *
 * @author madsfalken
 */
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Track {
    private String timestamp;
    private Location location;
    private Location pilotLocation;
    private double altitudeAGL;
    private String source;
}

File: utm-service/utm-service/Models/Track.cs
using System;
namespace utm_service.Models
{
    public class Track
    {
        public Coordinates location { get; set; }
        public string timestamp { get; set; }
        public double altitudeAGL { get; } = 3.5;
        public string source { get; } = "simulator";
    }
}
